development/build/index.js

```diff
@@ -3,6 +3,10 @@
 //
 // run any task with "yarn build ${taskName}"
 //
+global.globalThis = global
```

Comment:
I'm seeing this lint error:
```
6:1  error  The 'globalThis' is not supported until Node.js 12.0.0. The configured version range is '^10.16.0'  node/no-unsupported-features/es-builtins
```

The update to `@metamask/eslint-config` brought in a few new lint rules, including `node/no-unsupported-features/es-builtins`. I'd suggest ignoring it for this line.